src/scss/settings/_variables.scss

```diff
@@ -9,7 +9,7 @@
 // Set this in pixels (but do not add px),
 // the font-size mixin will convert to REMS
 
-$font-size-base             : $font-paragraph-02;
+$font-size-base             : $font-paragraph-01;
```

Comment:
@dandel10n I noticed that in the typography module, where this variable is used, [the body styles override this font-size increase and set them back to 14px](https://github.com/justeat/fozzie/blob/24cc930c798a4e5a97563b6bbe21f52d214ed022/src/scss/base/_typography.scss#L33).

Reply:
Thanks, Damian, I missed this, I will make a PR to change this :+1: